licenses.yaml

```diff
@@ -1668,1285 +1604,1459 @@ libraries:
+  - com.fasterxml.jackson.module: jackson-module-guice
 
 ---
 
-name: react-dom
+name: Google APIs Client Library For Java
 license_category: binary
-module: web-console
-license_name: MIT License
-version: 16.8.6
-copyright: Facebook, Inc. and its affiliates
-license_file_path: licenses/bin/react-dom.MIT
+module: java-core
+license_name: Apache License version 2.0
+version: 1.22.0
+libraries:
+  - com.google.api-client: google-api-client
 
 ---
 
-name: react-is
+name: Google HTTP Client Library For Java
 license_category: binary
-module: web-console
-license_name: MIT License
-version: 16.8.6
-copyright: Facebook, Inc. and its affiliates
-license_file_path: licenses/bin/react-is.MIT
+module: java-core
+license_name: Apache License version 2.0
+version: 1.22.0
+libraries:
+  - com.google.http-client: google-http-client
+  - com.google.http-client: google-http-client-jackson2
 
 ---
 
-name: react-lifecycles-compat
+name: Google OAuth Client Library For Java
 license_category: binary
-module: web-console
-license_name: MIT License
-version: 3.0.4
-copyright: Facebook, Inc.
-license_file_path: licenses/bin/react-lifecycles-compat.MIT
+module: java-core
+license_name: Apache License version 2.0
+version: 1.22.0
+libraries:
+  - com.google.oauth-client: google-oauth-client
+
+---
+
+name: JavaBeans Activation Framework API
+license_category: binary
+module: java-core
+license_name: CDDL 1.1
+version: 1.2.0
+copyright: Oracle and/or its affiliates
+license_file_path: licenses/bin/javax.activation-api.CDDL11
+libraries:
+  - javax.activation: javax.activation-api
 
 ---
 
-name: react-popper
+name: Apache Kafka
 license_category: binary
-module: web-console
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 0.8.2.1
+libraries:
+  - org.apache.kafka: kafka_2.10
+  - org.apache.kafka: kafka-clients
+
+---
+
+name: ZooKeeper Client
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 0.3
+libraries:
+  - com.101tec: zkclient
+
+---
+
+name: Metrics Core Library
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 2.2.0
+libraries:
+  - com.yammer.metrics: metrics-core
+
+---
+
+name: JOpt Simple
+license_category: binary
+module: extensions/kafka-extraction-namespace
 license_name: MIT License
-version: 1.3.3
-copyright: Travis Arnold
-license_file_path: licenses/bin/react-popper.MIT
+copyright: Paul R. Holser, Jr.
+version: 3.2
+license_file_path: licenses/bin/jopt-simple.MIT
+libraries:
+  - net.sf.jopt-simple: jopt-simple
+
+---
+
+name: Scala Library
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: BSD-3-Clause License
+copyright: EPFL, Lightbend Inc.
+version: 2.10.4
+license_file_path: licenses/bin/scala-lang.BSD3
+libraries:
+  - org.scala-lang: scala-library
 
 ---
 
-name: react-router-dom
+name: "@babel/polyfill"
```

Comment:
If you think it's a good idea to keep `web-console` dependencies separately from the Java stuffs, then it would be nice to add a comment to start a new section like `# Licenses for web console dependencies`.